### ISSUE TITLE
Incompatibility with brand new tensorflow version 2.4.0

### ISSUE THREAD

**Issue by felixdivo** [tslearn-team/tslearn#320]
Seems like it's the new version of tensorflow: [2.4.0](https://github.com/tensorflow/tensorflow/releases/tag/v2.4.0). Previously, `tensorflow-2.3.1` was installed instead.

The errors can be inspected in [this](https://dev.azure.com/romaintavenard/tslearn/_build/results?buildId=670&view=logs&j=2e703997-b85c-5955-fe15-cdf93bef6757&t=b1b94f87-1d38-5554-e373-8f4cd3c9d658&l=365) CI job:

```
FAILED tslearn/shapelets/shapelets.py::tslearn.shapelets.shapelets.GlobalMinPooling1D
FAILED tslearn/tests/test_shapelets.py::test_shapelet_lengths - AssertionError: 
```

**Comment by felixdivo** [comment]
See https://github.com/tslearn-team/tslearn/pull/319#issuecomment-745141246

**Comment by felixdivo** [comment]
Will be fixed by @rtavenar in #273 after commit d8d01d2
